### PR TITLE
Added opening animation to lines in a chart

### PR DESCRIPTION
**Pull request by danlevings**
Added a prop to the LineChart called 'openWithAnimation'.

This should animate both the line and the area smoothly

### CODE REVIEW

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-121.surge.sh

File: src/components/Line/index.js [hunk 1]
```diff
@@ -2,103 +2,162 @@ import React from 'react';
...
+    return (
+      <g clipPath={`url(#${clipPath})`}>
+        {area && (
+          <path
+            className="line-area"
+            d={area(data)}
+            style={{
+              stroke: 'none',
+              strokeWidth: `${strokeWidth}px`,
+              transition: 'clip-path 2s ease',
```
Comment (evancharlton) [inline src/components/Line/index.js]: Don't we only want to do this if animations are set?

File: src/components/LineChart/index.js [hunk 2]
```diff
@@ -86,6 +89,7 @@ const defaultProps = {
   areas: [],
   onAreaDefined: null,
   onAreaClicked: null,
+  openWithAnimation: false,
```
Comment (evancharlton) [inline src/components/LineChart/index.js]: What about `animationDuration` being a number? 0/null/undefined/false = nothing

File: stories/index.js [hunk 3]
```diff
@@ -119,6 +119,18 @@ storiesOf('LineChart', module)
       </DataProvider>
     ))
   )
+  .add(
+    'Basic with animation',
```
Comment (evancharlton) [inline stories/index.js]: Could you add an animated min/max example, too?

**Comment by danlevings** [comment]
Changed prop to 'openingAnimationDuration', improved stories to customise speed, fixed weird lines to the right - enjoy!

**Comment by danlevings** [comment]
Technically, thats right - you could probably set this up to be outside of griff entirely.
Although imo it feels like it would be part of a new griff add-on for animations and other flashy effects

**Comment by danlevings** [comment]
So is it decided that we should scrap this PR and move it inside opint?

**Comment by evancharlton** [comment]
I would very much prefer this live in the client app for the time being -- but if there's anything you need (css classes, eg) to make it happen, please send a PR or file an issue.

**Comment by danlevings** [comment]
This will be implemented externally from griff (https://github.com/cognitedata/operational-intelligence/pull/1078) - closing

**Pull request closed without merging**